en/docs/references/error-catalog.md

```diff
@@ -216,3 +216,78 @@ This document describes all the REST API error codes that are used in WSO2 Ident
   </tbody>
 </table>
 </div>
+
+## Human Task Approval Errors
+
+<div class="errorcode">
+<table>
+  <thead>
+    <th>Error Code</th>
+    <th>HTTP Status Code</th>
+    <th>Error Message</th>
+    <th>Possible Cause</th>
+  </thead>
+  <tbody>
+      <tr>
+        <td>HTA-10001</a></td>
+        <td>403</td>
+        <td>Access Denied.</td>
+        <td>You are not authorized to perform this task.</td>
```

Comment:
Only the first letter of the first word of the error message needs to be a capital letter. The rest of the words in the error message can start with simple letters. So in this case, can we change this to "Access denied" and the next one to "Invalid input provided" please?

Reply:
updated